06.bowling_object/frame.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  def initialize(index, first_shot, second_shot = nil, third_shot = nil)
+    @index = index
+    @shots = [first_shot, second_shot, third_shot].compact
+  end
+
+  def index
+    @index
+  end
+
+  def calc_total_score(frames)
+    next_frame = frames[index + 1]
+    after_next_frame = frames[index + 2]
```

Comment:
```suggestion
    next_frame, after_next_frame = frames[index, 2]
```

こんな書き方でもいけそうです。（好みですが）

Reply:
このようなすっきりした書き方ができるんですね！
早速使ってみました。
勉強になりました。ありがとうございます。